### DISCUSSION TITLE
PAZ data processing with ISCE2

### DISCUSSION THREAD

**Discussion by svenborgstrom** [isce-framework/isce2#401] [Q&A]
Good morning,
this is just to know if ISCE is currently supporting (or going to) PAZ data processing.
PAZ is the Spanish twin of TSX, which is fully supported by ISCE.
I've slightly changed the "unpackFrame_TSX" script to get the SLC in an ISCE readable format and it works fine, but at the "stackStripMap" step I get the error message "the sensor is not hard-coded", besides a "frame" error message, with no "run_files" folder at the end.
As far as I can understand PAZ data processing would be not so different than TSX processing.
Thanks,
Sven.

**Comment by piyushrpt** [comment]
You will have to run the unpack scripts to create a folder with unpacked images organized by dates. I don't think there is any publicly available free data to test PAZ reader. If you have a reader that works, this should work. You really don't need sensor names for running the stacking script. Then use stackStripMap.py with the unpacked SLCs as input.

**Comment by krasny2k5** [comment]
Hi there:

I successfully processed some data from PAZ using a modified version of the TSX unpackFrame script. I used it with the uncompressed folders of PAZ and always using the full path for the input files instead of relative paths. Let me know if you have any problems.

@piyushrpt I'm not aware if there is any public data of PAZ, but I will ask the science department to see if there is any chance to release some data as an example. The internal structure of the data is almost identical to the TSX and it seems that is pre-processed with the same software as the DLR ones.

[unpackFrame_PAZ.zip](https://github.com/isce-framework/isce2/files/7597811/unpackFrame_PAZ.zip)
u.

**Reply by piyushrpt** [reply]
:+1:  Best approach might be to issue a PR with the PAZ reader.

**Reply by svenborgstrom** [reply]
Hi everybody,
at the end I succeeded: I used my script to get SLCs, then "stackStripMap" without the sensor name in the script  (just the azimuth and range values for multilooking) as suggest by Piyush.
I got PAZ data in the frame of an AO for PAZ data request over La Palma, following the recent eruption, b.t.w. I could ask the INTA (Instituto Nacional de Tecnica Aeroespacial) colleagues if they can provide a PAZ scene if the ISCE developers want to do some test and let's see how they answer.
Thanks everybody for your support.
Sven.

**Reply by fdelgadodelapuente** [reply]
Dear Sven,

It would be great if you could ask INTA to release and share the PAZ data for testing it with ISCE.

Francisco
